app/models/observation.rb

```diff
@@ -169,9 +169,9 @@ class Observation < AbstractModel
                      through: :observation_views,
                      source: :user
 
+  before_destroy { destroy_orphaned_collection_numbers }
```

Comment:
I don't know why this moved! I assumed it was from an earlier commit on master, but it seems not. Feel free to move it back.

Reply:
Also don't know why it uses bracket syntax instead of symbol. Going to commit the move back to old position now.